compiler/src/dotty/tools/dotc/parsing/Scanners.scala

```diff
@@ -736,7 +736,10 @@ object Scanners {
                  && currentRegion.commasExpected
                  && (token == RPAREN || token == RBRACKET || token == RBRACE || token == OUTDENT)
               then
-                () /* skip the trailing comma */
+                // encountered a trailing comma
+                // reset only the lastOffset
+                // so that the Ident's span is correct
```

Comment:
I don't think this is specific to `Ident`s so maybe just say `so that the span is correct`